### PR TITLE
updated the styling on the sync your discord to egghead page

### PR DESCRIPTION
**Pull request by lsminter**
Since I'm sending out the `/discord` link to our learners as we are interviewing them, I figured it would be nice to send them a halfway decent-looking page. 

Old version:
Screen Shot 2022-03-18 at 1 07 04 PM

New version:

new version

### CODE REVIEW

File: src/pages/discord/index.tsx [hunk 1]
```diff
@@ -8,8 +8,8 @@ const Discord: FunctionComponent<LoginRequiredParams> = ({loginRequired}) => {
   return (
     <LoginRequired loginRequired={loginRequired}>
       {process.env.NEXT_PUBLIC_DISCORD_AUTHORIZE && (
-        <div className="w-1/2 flex space-y-6   flex-col">
-          <p className="prose lg:prose-xl">
+        <div className="grid place-items-center h-96">
+          <p className="prose lg:prose-xl text-white text-center">
```
Comment (lsminter) [inline src/pages/discord/index.tsx:12]: Changed to `grid` from `flex` to better center the items on the page, changed the text to white so it was easier to read.

**Comment by lsminter** [comment]
Just realized I haven't checked for light mode. I need to do that before merging.

File: src/pages/discord/index.tsx [hunk 2]
```diff
@@ -9,7 +9,7 @@ const Discord: FunctionComponent<LoginRequiredParams> = ({loginRequired}) => {
     <LoginRequired loginRequired={loginRequired}>
       {process.env.NEXT_PUBLIC_DISCORD_AUTHORIZE && (
         <div className="grid place-items-center h-96">
-          <p className="prose lg:prose-xl text-white text-center">
+          <p className="prose lg:prose-xl dark:text-white text-center">
```
Comment (lsminter) [inline src/pages/discord/index.tsx:12]: Now it works for both dark and light mode.

**Comment by vojtaholik** [comment]
I applied some design treatment :) https://github.com/eggheadio/egghead-next/pull/1077/commits/b70a6ac07f236b7b562e086a7697c2b337de9417

screenshots

**Review by vojtaholik — APPROVED** [review]

**Pull request merged**